### PR TITLE
Update node label

### PR DESCRIPTION
**Pull request by llxia**
Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
Reviewers: @ShelleyLambert @AdamBrousseau

**Comment by llxia** [comment]
Following label does not have match. Please update the machine labels. Thanks.
https://ci.adoptopenjdk.net/label/sw.os.linux&&hw.arch.s390x/
https://ci.adoptopenjdk.net/label/hw.arch.ppc&&sw.os.aix/ - please ignore, there is no test machine

https://hyc-runtimes-jenkins.swg-devops.com/label/sw.os.linux&&hw.arch.ppcle/

**Comment by AdamBrousseau** [comment]
ref: AdoptOpenJDK/openjdk-infrastructure#93

File: buildenv/jenkins/openjdk10_ppc64_aix [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	Set parameter TARGET(openjdk, system, external, perf, jck etc.).
 	Set UPSTREAM_JOB_NAME(version_build_arch_os, for example: openjdk8_build_x86-64_linux)
 	Set JVM_VERSION(openjdk8, openjdk8-openj9, openjdk9, openjdk8-openj9, etc.*/
-LABEL='aix&&ppc64&&test||ci.project.openj9&&hw.arch.ppc&&sw.os.aix7.1'
+LABEL='aix&&ppc64&&test||hw.arch.ppc&&sw.os.aix'
```
Comment (smlambert) [inline buildenv/jenkins/openjdk10_ppc64_aix]: btw if you want to get rid of the aix&&ppc64&&test label, I can tag the aix build machines appropriately (for the [weekly builds that we are running](https://ci.adoptopenjdk.net/view/Weekly%20Tests/), so you can simplify further)

Comment (llxia) [reply buildenv/jenkins/openjdk10_ppc64_aix]: removed `aix&&ppc64&&test`.  @smlambert could you merge this PR once you update the label? Thanks.

**Pull request merged**